app/views/admin/publishers/show.html.slim

```diff
@@ -18,11 +18,19 @@
           .db-field = "Status:"
           .db-value = publisher_status(@publisher)
         .db-info-row
-          .db-field = "Uphold verified:"
+          .db-field = "Uphold once connected:"
           .db-value = @publisher.uphold_verified?
         .db-info-row
           .db-field = "Uphold authorized:"
           .db-value = @publisher.wallet.authorized?
+        - if @publisher.wallet.status.present?
+          .db-info-row
+            .db-field = "Uphold status:"
+            .db-value = @publisher.wallet.status
```

Comment:
IIRC this isn't necessarily the status we would want. I was getting `re-authorize` for my account, so I believe this comes from Eyeshade/Ledger and not from Uphold.